packages/core/src/2d/text/TextUtils.ts

```diff
@@ -247,68 +247,12 @@ export class TextUtils {
   }
 
   static async registerFont(fontName: string, fontUrl: string): Promise<boolean> {
-    if (FontFace) {
-      return this._registerFontWithFontFace(fontName, fontUrl);
-    } else {
-      return this._registerFontWithCSS(fontName, fontUrl);
-    }
-  }
-
-  private static async _registerFontWithFontFace(fontName: string, fontUrl: string): Promise<boolean> {
     const fontFace = new FontFace(fontName, `url(${fontUrl})`);
```

Comment:
if `FontFace` API not exists, you should use default system font instead. The downgrade logic should be added.